docs/configurations/kafkacluster/examples/_index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Kafka cluster
+shorttitle: KafkaCluster
+weight: 7000
+---
+
+We created KafkaCluster custom resource examples for some basic use-case.  
+You can use these examples as a base to your own Kafka cluster.  
+
+### KafkaCluster CR with detailed explanation
```

Comment:
**Note**: I would give a bit more thought to the formatting here.
I'm somewhat okay with the current one, but don't necessarily feel like it just clicks the right way.

I was thinking about maybe proposing a table based solution where we could use sections (separator row delimited rows) for multiple examples groups while single lines would be enough for single example topics.
We could start with the name which could be the link to the configuration itself, then the next cell in the row could describe the most important Koperator features used in the configuration example, such as ACL, SSL, etc, then the next cell in the row could give a description.
That is just a shower thought, feel free to improve upon those.

I don't necessarily have a strong opinion on this, just feel like there is room to improve here.